integration-tests/utils/pause.ts

```diff
@@ -0,0 +1,3 @@
+export const pause = (delay: number): Promise<unknown> => {
```

Comment:
I think this could actually live in the `helpers.ts` file instead of its own file.